### DISCUSSION TITLE
SU2_AD Installtion

### DISCUSSION THREAD

**Discussion by Adiithyaa** [su2code/SU2#2299] [Q&A]
Hello , I am new to using SU2.My SU2_AD hasn't been installed and when I run the command given in the website :
export CXXFLAGS="-O3 -Wall" && ./preconfigure.py --enable-autodiff --enable-mpi --prefix=$SU2_INSTALLPATH
,it shows the following error:
preconfigure.py: error: unrecognized arguments: --enable-autodiff --enable-mpi --prefix=
Please help me out

**Comment by joshkellyjak** [comment]
These commands are for an older version of SU2. You should follow the instructions in the v7+ documents if you are using a more recent version.

**Reply by Adiithyaa** [reply]
Can you mention which document it is or the code needed to be used because I couldn't find the required document

**Reply by TobiKattmann** [reply]
Information concerning the build are here: https://su2code.github.io/docs_v7/Build-SU2-Linux-MacOS/
